### PR TITLE
Change the release jobs to use the second aws account

### PR DESCRIPTION
**Pull request by droslean**
/hold
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Comment by droslean** [comment]
/cc @wking @e-tienne FYI. Can you check the results of the rehearsals? 
@wking  Which branches we should migrate to the 2nd aws account? IIRC >4.5 ?

**Comment by e-tienne** [comment]
> /cc @wking @e-tienne FYI. Can you check the results of the rehearsals?

At first sight, it looks like a DNS issue. How are we handling CI DNS zone in the new AWS account? If it's a new one such as '_origin-ci-int-aws-2.dev.rhcloud.com_' we would need to make sure it's configured in the `install-config.yaml` file that we're using for jobs in this new account.

`level=error msg=Bootstrap failed to complete: Get "https://api.ci-op-n1cz2n46-316a0.origin-ci-int-aws.dev.rhcloud.com:6443/version?timeout=32s": dial tcp: lookup api.ci-op-n1cz2n46-316a0.origin-ci-int-aws.dev.rhcloud.com on 172.30.0.10:53: no such host`

**Comment by droslean** [comment]
@wking @e-tienne If we exclude the VPC limit error, I think we are in a good shape. Can you check?

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.5.yaml [hunk 1]
```diff
@@ -64,7 +64,9 @@ tests:
 - as: e2e-aws-proxy
   cron: 0 0 */2 * *
   steps:
-    cluster_profile: aws
+    cluster_profile: aws-2
+    env:
+      BASE_DOMAIN: aws-2.ci.openshift.org
```
Comment (e-tienne) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.5.yaml:69]: This fixes the DNS issue we saw at first.

**Review by e-tienne — COMMENTED** [review]
I can't see AWS or cloudformation logs right now, but I would want to see at least `openshift-e2e-aws-proxy` passing.

**Comment by e-tienne** [comment]
> @wking @e-tienne If we exclude the VPC limit error, I think we are in a good shape. Can you check?

Can you please confirm if the cloudformation stack errors I see in the logs (of `openshift-e2e-aws-proxy`) are related to this?

**Comment by droslean** [comment]
/hold cancel

@petr-muller @wking Shall we merge this?

**Comment by petr-muller** [comment]
@droslean please remove all the `proxy` and `e2e-aws-workers-rhel7` jobs from the PR, they aren't AWS-2 cabable. Then we can merge it.
/hold

**Comment by droslean** [comment]
@petr-muller Updated.

**Review by petr-muller — APPROVED** [review]
/hold cancel

**Pull request merged**